ansible/playbooks/roles/upgrade/tasks/kubernetes/patch-kubeadm-image-repository.yml

```diff
@@ -2,7 +2,7 @@
 - name: k8s/master | Patch imageRepository in kubeadm-config ConfigMap
   run_once: true
   block:
-    - name: k8s/master | Get kubeadm-config configmap
+    - name: k8s/utils | Get kubeadm-config configmap
```

Comment:
The next task has different prefix. I would prefer to use the same in across the file (unless task is delegated).

Reply:
This is a mistake left after trying moving that part into a separate file.